pkg/controller/util/overrides.go

```diff
@@ -27,8 +28,9 @@ import (
 )
 
 type ClusterOverride struct {
+	Op    string      `json:"op,omitempty"`
 	Path  string      `json:"path"`
-	Value interface{} `json:"value"`
+	Value interface{} `json:"value,omitempty"`
```

Comment:
I'm assuming the reason for it being optional is that `interface{}` is implicitly a pointer.  I don't think there is much value in worrying about this struct since it is just a convenience to enable typed use rather than a public interface.  I share your concern about the validation schema, though - these fields really should be required.